ci/gpu/build.sh

```diff
@@ -61,6 +61,11 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "rapids-doc-env=${MINOR_VERSION}.*" \
       "shap>=0.37,<=0.39"
 
+gpuci_conda_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      treelite=2.0.0
+
```

Comment:
With CI passing and https://github.com/rapidsai/integration/pull/315 merged, we can revert these changes now and prepare this PR to be merged. It will likely take a few hours for the changes in https://github.com/rapidsai/integration/pull/315 to be propagated to CI, however.